symbolic_pymc/tensorflow/printing.py

```diff
@@ -41,8 +42,16 @@ def println(self, obj):
         self.buffer.write(self.format(obj) + "\n")
         self.buffer.flush()
 
+    def depth_check(self, current_depth):
+        if isinstance(self.depth_index, (list, tuple, range)):
+            return current_depth in self.depth_index
+        elif self.depth_index is None:
+            return True
+        else:
+            raise ValueError("Depth needs to be a tuple, list or range type.")
+
 
-def tf_dprint(obj, printer=None):
+def tf_dprint(obj, printer=None, index=None):
```

Comment:
This needs a good explanation of what `index` is/does, especially when it's a collection.

Reply:
I renamed `index` to `depth_index` to make it more clear what it does.